protocols/noise/Cargo.toml

```diff
@@ -16,7 +16,7 @@ log = "0.4"
 prost = "0.6.1"
 rand = "0.7.2"
 ring = { version = "0.16.9", features = ["alloc"], default-features = false }
```

Comment:
```suggestion
```

Reply:
We're currently only using `ring` directly for SHA256. Should i switch the code to the `sha2` crate?